### PR TITLE
Added trials project

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description

-Edited Read me file
-Reviewed Microverse trials project.
-Added 2 comments to project.

# Screenshots

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>Fork Me! FCC: Test Suite Template</title>
```
Comment (joshualyngdoh) [inline index.html:5]: Kindly, I suggest you indicate a suitable "title" for your web page.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  #microverse{
+     text-align: center;
+  }
+  
+  #header-img{
+    width:40%;
+  }
+  #video{
+    margin-top: 60px;
+
```
Comment (joshualyngdoh) [inline css/style.css:35]: Kindly remove spaces to make for a cleaner code.

**Review by joshualyngdoh — COMMENTED** [review]
# Highlights


- Correct Gitflow flow.
- Meaningful commit messages.

# Required changes

- Kindly set up the linters in your project please add them in order to set up [follow this](https://github.com/microverseinc/linters-config/tree/master/html-css). The main purpose of this project is to set up the linters.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @joshualyngdoh,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Professional and polite comments. ✔️
 - Good use of inline comments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**